integration/test/testConfig.js

```diff
@@ -0,0 +1,43 @@
+const devApiBaseUrl = 'https://dev-api.clearlydefined.io'
+const prodApiBaseUrl = 'https://api.clearlydefined.io'
+
+const pollingInterval = 1000 * 60 * 5 // 5 minutes
+const pollingMaxTime = 1000 * 60 * 30 // 30 minutes
+
+const harvestToolVersions = [
+  ['licensee', '9.14.0'],
+  ['scancode', '30.3.0'],
+  ['reuse', '3.2.1']
```

Comment:
@lumaxis @yashkohli88, after scan tool update, the versions here will need to be updated.